addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -332,6 +328,41 @@ private String convertToText(JsonElement element) {
         // }
     }
 
+    private String convertToImageUrl(JsonElement element) {
+        String text = convertToText(element);
+        if (!text.isEmpty()) {
```

Comment:
Can it be null?

Reply:
I doubt that. `convertToText` returns the `getAsString()` value of the given `JsonElement`. And the given `JsonElement` cannot be `null` because we check if it exist before accessing it (see https://github.com/openhab/openhab2-addons/pull/2304/files#diff-cf13d584f28ada8d7a87e83665490af0R277)

Reply:
👍